### PR TITLE
Auto refresh

### PR DESCRIPTION
**Pull request by GuusK**
This pull request includes a

- [ ] Bug fix
- [x] New feature
- [ ] Translation

The following changes were made

- Added global variable and flag to set it
- Added multiple REST calls to allow for AJAX request
- Added Jquery and AJAX request to update the map

### CODE REVIEW

**Comment by GuusK** [comment]
Sorry for the mess with git. I tried to fix it and pull the changes of the master into my fork and then into the branch, but it seems to have failed

**Comment by DerScreever** [comment]
Nah no problem. ^^

**Comment by nborrmann** [comment]
Looking at the code I really like it. However untested as the servers are down for me right now.

There will be conflicts with #101 and #104 though.

**Comment by favll** [comment]
Just a few minor remarks concerning the command line option:

- change `--auto_refresh` to `--auto-refresh` to maintain consistency (dashes instead of underscores in cl arguments)
- maybe provide a default value (30 seconds maybe)?

Also I would change the `/getData` route to simply `/data` and the `/getConfig` route to `/config`, etc. Since this is a REST api, the verb of the operation is already part of the HTTP method (i.e. `GET /data HTTP/1.1`). 

Furthermore, change camelCase method names to snake_case to adhere to python style guide; `getConfig()` to `get_config`, etc.

Finally, Flask's `jsonfiy` method should be used to return json data. The added benefits are listed [here](http://flask.pocoo.org/docs/0.11/api/#flask.json.jsonify); e.g. change `return json.dumps(center)`, etc.

File: templates/example_fullmap.html [hunk 1]
```diff
@@ -5,12 +5,69 @@
...
+        $.get(baseURL + "/getConfig", function(response){
+                        var json_obj = $.parseJSON(response);//parse JSON
+                        options["lat"] = json_obj["lat"];
+                        options["lng"] = json_obj["lng"];
+                        options["zoom"] = json_obj["zoom"];
+                        options["identifier"] = json_obj["identifier"];
+                    });
+
+        window.setInterval(function(){
+            // A new map is created because the original one isn't saved
```
Comment (rochacbruno) [inline templates/example_fullmap.html]: Np need to create a new map! now it is possible to reference `map` object direclty

**Pull request closed without merging**